src/classes/DailyHabitTaskStatusResetToNew.cls

```diff
@@ -0,0 +1,19 @@
+global class DailyHabitTaskStatusResetToNew implements Schedulable {
+
+
+	global void execute(SchedulableContext sc) {
```

Comment:
You should not perform these actions from a schedulable class. You should use this class in order to execute at a certain time and simply call a Batchable class from here.

Reply:
Done